corehq/apps/reports/standard/users/reports.py

```diff
@@ -179,15 +177,39 @@ def _html_list(changes, unstyled=True):
 
     properties = UserHistoryReport.get_primary_properties(domain)
     properties.pop("user_data", None)
-    primary_changes = {
-        properties.get(key, key): value for key, value in changes.items()
-        if key in properties
-    }
-    more_count = len(changes) - len(primary_changes)
+    primary_changes = {}
+
+    for key, value in changes.items():
+        if key in properties:
+            if key == 'commcare_location_id':
```

Comment:
isn't this "location_id"?

Reply:
i see you have used location_id later. 
My understanding is that location_id is set on the user itself for mobile workers and on domain membership for web users. The log for now, will have "location_id" present for commcare users, i am not sure where "commcare_location_id" is coming from.

Reply:
`commcare_location_id` is set in the user's metadata so it can be accessed in apps.  It should always be the same as location_id:
https://github.com/dimagi/commcare-hq/blob/49e3b311c061704114c9b008b09de52afe1b5705/corehq/apps/users/models.py#L1984-L1984